troubleshooting.md

```diff
@@ -746,3 +746,141 @@ an Infra container image for CNI-in-slirp4netns must be created.  The
 instructions for building the Infra container image can be found for
 v2.2.1 [here](https://github.com/containers/podman/tree/v2.2.1-rhel/contrib/rootless-cni-infra),
 and for v3.0.1 [here](https://github.com/containers/podman/tree/v3.0.1-rhel/contrib/rootless-cni-infra).
+
+### 29) Container related firewall rules are lost after reloading firewalld
+Container network can't be reached after `firewall-cmd --reload` and `systemctl restart firewalld` 
```

Comment:
Lint is complaining there's trailing whitespace on this line - remove the extra space at the end and CI will pass.

Reply:
Hey, I removed the trailing whitespace and pushed the commit that should do it, hopefully.

Reply:
I think I messed up badly with the git commits my git log is also not in order for some reason.